models/userlocation.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
So I think strictly speaking, you don't need to write this model. Sequelize auto generates it when you add the belongsToMany in User and Location. I don't think it matters anyway but just fun fact

Reply:
Oh cool. Thanks!